packages/ui/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../tsconfig.base", // not set up yet
-  "include": ["src"],
+  "include": ["src", "../../apps/expo/app/Bugsnag.ts"],
```

Comment:
Were you getting a type error on this ?

Reply:
This appears to be an automated add-on, I think.